node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,13 @@
+import { readFile } from 'node:fs/promises';
+
+async function readAny(file) {
+  try {
+    const filePath = './' + file;
+    const contents = await readFile(filePath, { encoding: 'utf8' });
```

Comment:
As a shortcut you can just pass 'utf8' as the second parameter:
`readFile(filePath, 'utf8');`